modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -2605,22 +2605,6 @@ private void processSymbolizers(final Graphics2D graphics,
         }
     }
 
-    private Geometry clip(Geometry geometry, final GeometryClipper clipper,
-            boolean preserveTopology) {
-        try {
-            return clipper.clip(geometry, !preserveTopology);
```

Comment:
As a side effect, I think the map is drawing quicker!  I don't have great metrics, but I think it is.